tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/util/ServerUtil.java

```diff
@@ -32,6 +32,7 @@
 import com.liferay.ide.server.remote.IServerManagerConnection;
 
 import java.io.File;
+import java.io.FileInputStream;
```

Comment:
Did you format extra lines in this file?  Please restore them so it is easier for me to see what you changed here.  Thanks.